src/pages/YXWPRedux/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import { Button, Card, Input, Table, Space } from 'antd'
+import { ShoppingHeader } from './styles/YXWPRedux.styled'
+import { PlusCircleOutlined, MinusCircleOutlined } from '@ant-design/icons'
+import { useDispatch, useSelector } from 'react-redux'
+import { increaseQuantity, decreaseQuantity, deleteItem, addItem } from '../../store/yxwp_store/actions'
+import { RootState } from '../../store/types'
+import { ShoppingListEntry } from '../../store/yxwp_store/types'
+const { Column } = Table
+
+const YXWPRedux = () => {
+  const dispatch = useDispatch()
+  const { itemList } = useSelector((state: RootState) => state.yxwp_store)
+  const blankItem: ShoppingListEntry = { itemName: '', itemQuantity: 0, key: -1 }
```

Comment:
Setting itemQuantity to 0 when quantity of item cannot be decreased below 1.

Small issue, can ignore.

Reply:
Fixed in commit b274fd2